can/CAN.py

```diff
@@ -1,9 +1,13 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
The shebang is only useful if the file can be executed as a script. Most files here are modules so it doesn't add any value.

I don't think there are any non-ASCII characters in these files so the UTF-8 should not be needed either. Besides, UTF-8 is default in Python 3.

Reply:
These changes originate from a different PR that this one was rebased on, since it required changes that were made over there. That is also the reason for why there are so many file that were changed. 

I think the shebang does add value as it make clear that it is compatible with Python 2 & 3 as well as being uniform. But I guess that's subjective. It doesn't do any harm however.

The `coding` line does add value as well, since it enforces the same behavior on Python 2 & 3. That is desirable since everything that makes these versions more alike in such manners is welcome for maintainability. It also makes it explicit.